### ISSUE TITLE
SPFx generator references "clientsideinstance.xml" when file is sentence case (fails build on Linux)

### ISSUE THREAD

**Issue by jhueppauff** [SharePoint/sp-dev-docs#4005]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

I am not sure if this is the right Repo to report this issue.
I tried to create a new spfx extension using the yo generator @microsoft/generator-sharepoint

The generator includes this File: ```package-solution.json```
Within this file the ```ClientSideInstance.xml``` is written as lower case.
If you build this solution on a Linux based system (e.g. Azure DevOps Build using Ubuntu as this is the default for NPM Builds), you will run into an error

```
Found Element Manifest: elements.xml
Error - 'package-solution sub task errored'
Could not find Element Manifest: sharepoint/assets/clientsideinstance.xml
```

If you alter the elements.xml using the exact file name, the build will be successful.

```
"assets": {
          "elementManifests": [
            "elements.xml",
            **"ClientSideInstance.xml"**
          ]
        }

```
#### Steps to Reproduce

1) Install Sharepoint Generator for yo (install yo if not installed)
`npm install @microsoft/generator-sharepoint -g`

2) Create a new extention using the yo SharePoint Generator
`yo @microsoft/sharepoint --solution-name "solution-name"  --component-type "extension" --component-description "List Command Bar Extention" --environment "spo" --extension-type "ListViewCommandSet" -- skip-feature-deployment  --component-name "component-name"`

3) Within the `elements.xml` the filename will be lower case. Builds on Linux Systems will fail.

**Comment by andrewconnell** [comment]
Good catch... renamed the issue to be more specific on what's going on...

FYI - @patmill @VesaJuvonen, flagged for next SPFx drop?

**Comment by lucabandMSFT** [comment]
@andrewconnell. assigned to dev for triage... will let you know soon if we can make it for the next drop ;-)

**Comment by KevinTCoughlin** [comment]
This fix has been merged and will be available in next SPFx drop cc: @lucabandMSFT.

**Comment by johnguy0** [comment]
Fixed in `@microsoft/generator-sharepoint@1.10.0`
